gcs/src/dashboard.jsx

```diff
@@ -170,6 +170,7 @@ export default function Dashboard() {
   // Map and messages
   const mapRef = useRef()
   const [outsideVisibility, setOutsideVisibility] = useState(false)
+  var outsideVisibilityColor = outsideVisibility ? '#171A1B' : 'rgba(23, 26, 27, 0.8)'
```

Comment:
Can you change the hex to the RGB

Reply:
we could change it to use tailwind colours but I can't use "bg-falcongrey-xxx" here as it needs to directly use the style prop

Reply:
Nah just keep it rgb, but change it so it's ` var outsideVisibilityColor = outsideVisibility ? 'rgb(23, 26, 27)' : 'rgba(23, 26, 27, 0.8)'`

Reply:
but its related to falcongrey120 so it needs to use that if its updated

Reply:
new push has moved everything to hex using tailwind config colours

Reply:
Can you not do  var outsideVisibilityColor = outsideVisibility ? tailwindColors[falcongrey][120] : 'rgba(23, 26, 27, 0.8)'?

Reply:
I did, but changed it to a better method